filebeat/module/nginx/ingress_controller/ingest/pipeline.yml

```diff
@@ -295,6 +295,10 @@ processors:
       field: related.ip
       value: "{{destination.ip}}"
       if: "ctx?.destination?.ip != null"
+  - append:
+      field: related.ip
+      value: "{{nginx.ingress_controller.upstream.ip}}"
+      if: "ctx?.nginx?.ingress_controller?.upstream?.ip != null"
```

Comment:
please add `allow_duplicates: false`, similar to the integration - https://github.com/elastic/integrations/blob/main/packages/nginx_ingress_controller/data_stream/access/elasticsearch/ingest_pipeline/default.yml#L297-L301

Reply:
@tetianakravchenko Thanks for the helpful advice! Done.